easybuild/easyconfigs/a/AMOS/AMOS-3.1.0_link_correctly_to_DELTAFILER_SHOWCOORDS.patch

```diff
@@ -0,0 +1,14 @@
+minimus2 needs delta-filter and show-coords but these were not set correctly like NUCMER
```

Comment:
@laraPPr Can you please add yourself as author in both patches, and then fix the checksums accordingly?

```suggestion
minimus2 needs delta-filter and show-coords but these were not set correctly like NUCMER
author: Lara Peeters (HPC-UGent)